### PR TITLE
High level explanation

### PR DESCRIPTION
**Pull request by ltetrel**
#13

### CODE REVIEW

File: source/OVERVIEW.rst [hunk 1]
```diff
@@ -21,13 +21,20 @@ Jupyter Notebooks and Markdown files into publishable content.
 Providing online-executable runtimes and a powerful data caching system, NeuroLibre makes a **groundbreaking preprint
 server where you can plant the seeds of living publications.**
 
+.. seealso:: If you would like to submit a NeuroLibre preprint, please read our `submission guidelines <https://docs.neurolibre.com/en/latest/SUBMIT.html>`_.
+
+As an author and before submitting, you:
+
+1. Create a github repository with notebooks that you would like to share.
+2. Make your work reproducible by using `binder configuration files <https://mybinder.readthedocs.io/en/latest/using/config_files.html>`_
+3. (optionnaly) Test your repository on `binder <https://mybinder.org/>`_.
```
Comment (agahkarakuzu) [inline source/OVERVIEW.rst:30]: IIRC, this is not optional for submission. We'll require at least one successful run before they can submit.

File: source/OVERVIEW.rst [hunk 2]
```diff
@@ -21,13 +21,20 @@ Jupyter Notebooks and Markdown files into publishable content.
 Providing online-executable runtimes and a powerful data caching system, NeuroLibre makes a **groundbreaking preprint
 server where you can plant the seeds of living publications.**
 
+.. seealso:: If you would like to submit a NeuroLibre preprint, please read our `submission guidelines <https://docs.neurolibre.com/en/latest/SUBMIT.html>`_.
+
+As an author and before submitting, you:
```
Comment (agahkarakuzu) [inline source/OVERVIEW.rst:26]: Before submitting: 

should be enough.

File: source/OVERVIEW.rst [hunk 3]
```diff
@@ -21,13 +21,20 @@ Jupyter Notebooks and Markdown files into publishable content.
...
+
+1. Create a github repository with notebooks that you would like to share.
+2. Make your work reproducible by using `binder configuration files <https://mybinder.readthedocs.io/en/latest/using/config_files.html>`_
```
Comment (agahkarakuzu) [inline source/OVERVIEW.rst:29]: Create a reproducible execution runtime for your notebooks:

File: source/OVERVIEW.rst [hunk 4]
```diff
@@ -21,13 +21,20 @@ Jupyter Notebooks and Markdown files into publishable content.
...
+3. (optionnaly) Test your repository on `binder <https://mybinder.org/>`_.
+4. Design a `Jupyter Book <https://jupyterbook.org/intro.html>`_ with all relevant bibliography.
```
Comment (agahkarakuzu) [inline source/OVERVIEW.rst:31]: Add Jupyter Book configurations and your bibliography:

File: source/OVERVIEW.rst [hunk 5]
```diff
@@ -44,4 +51,4 @@ Contributions are welcome, NeuroLibre is fully open-source
 
 NeuroLibre draws its strength from community-developed tools such as `BinderHub <https://github.com/jupyterhub/binderhub>`_ and `Open Journals <https://github.com/openjournals>`_.
 
-.. seealso:: If you are interested in contributing to NeuroLibre or to deploy one of you own, please visit `developer documentation <https://docs.neurolibre.com/en/latest/INFRASTRUCTURE.html`_.
\ No newline at end of file
+.. seealso:: If you are interested in contributing to NeuroLibre or to deploy one of you own, please visit `developer documentation <https://docs.neurolibre.com/en/latest/INFRASTRUCTURE.html>`_.
```
Comment (agahkarakuzu) [inline source/OVERVIEW.rst:54]: *one of your own

please visit NeuroLibre's developer documentation

**Review by agahkarakuzu — APPROVED** [review]
Thanks @ltetrel, added some comments.

File: source/OVERVIEW.rst (hunk 1 above)
Comment (ltetrel) [reply source/OVERVIEW.rst:30]: Hmm yeah I agree

File: source/OVERVIEW.rst (hunk 2 above)
Comment (ltetrel) [reply source/OVERVIEW.rst:26]: Note that this was your text haha. Anyway just changes it

File: source/OVERVIEW.rst (hunk 4 above)
Comment (ltetrel) [reply source/OVERVIEW.rst:31]: I did not want to make this list huge, and for me bibliogrpahy is part of the writing (i.e. jupyter book "design"). Can you think of something that takes no more than one line ?

**Comment by ltetrel** [comment]
Thanks for the review @agahkarakuzu !

**Pull request merged**